### PR TITLE
# fix device c height value

### PR DESCRIPTION
**Pull request by RedaHossino**
fix the height of device c with keeping the height type AMSL, for the new value, height AMSL 380 m, we have:
- altitude on the location of device c ~= 377 m
- haat of the cbd ~= 1.5 m 
- height AGL = 380 - 377 ~= 3 m

### CODE REVIEW

**Comment by jni2000** [comment]
Why not just simple change the heightType to "AGL" to make it consistent with all other device data ?

src/harness/testcases/testdata/device_a.json:    "heightType": "AGL",
src/harness/testcases/testdata/device_b.json:    "heightType": "AGL",
src/harness/testcases/testdata/device_c.json:    "heightType": "AMSL",
src/harness/testcases/testdata/device_d.json:    "heightType": "AGL",
src/harness/testcases/testdata/device_e.json:    "heightType": "AGL",
src/harness/testcases/testdata/device_f.json:    "heightType": "AGL",
src/harness/testcases/testdata/device_g.json:    "heightType": "AGL",
src/harness/testcases/testdata/device_h.json:    "heightType": "AGL",
src/harness/testcases/testdata/device_i.json:    "heightType": "AGL",

File: src/harness/testcases/testdata/device_c.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   "installationParam": {
     "latitude": 38.815291,
     "longitude": -97.198805,
-    "height": 4.5,
+    "height": 380,
     "heightType": "AMSL",
```
[the remark this answers is not in the document]
Comment (RedaHossino) [reply src/harness/testcases/testdata/device_c.json]: we proposed that in a previous PR 455, but Kate suggested to have at least one device with AMSL height type, which will allow to do tests  with a "correct" static Cbsd of AMSL height type .
don't you agree with that?

[the remark this answers is not in the document]
Comment (jni2000) [reply src/harness/testcases/testdata/device_c.json]: Ok, agreed, if that is the case.

**Comment by kate-harrison** [comment]
Please hold off on submitting -- I am still gathering the information requested yesterday.

**Comment by kate-harrison** [comment]
Based on today's discussion in WG1, we will introduce a minimum height into the ITM prop model and therefore (I think) we do not need to make this change. Do you agree?

**Comment by RedaHossino** [comment]
yes I agree with that.

**Pull request closed without merging**